.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 dist
 npm-debug.log
+.idea/
```

Comment:
👍 I use WebStorm as well. FYI you can always add this line to your local `.git/info/exclude` file to ignore it just for you, which is what I typically do when working with an open source project. But certainly no harm in adding this line to the .gitignore.